linux_os/guide/services/ssh/ssh_server/sshd_set_idle_timeout/rule.yml

```diff
@@ -42,13 +46,14 @@ references:
     cis: 5.2.12
     cjis: 5.5.6
     cui: 3.1.11
-    disa: 1133,2361
-    nist: AC-2(5),SA-8(i),AC-12,AC-17(b)
+    disa: 879,1133,2361
+    nist: AC-2(5),SA-8(i),AC-12,AC-17(b),MA-4(e),SC-10
```

Comment:
[MA-4(e)](https://nvd.nist.gov/800-53/Rev4/control/ma-4) is about terminating sessions when work is completed, not when idle timeouts occur. Can you help us understand the reasoning behind adding the MA-4(e) mapping?

Reply:
[SC-10](https://nvd.nist.gov/800-53/Rev4/control/sc-10) is perfectly applicable, thank you for adding that one in!